Riot/Assets/Images.xcassets/Room/Location/location_map_error.imageset/Contents.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Shouldn't we use SVG files since we support iOS 14?

Reply:
Yes you're right we can use SVG now